sdk/eventhub/Azure.Messaging.EventHubs/src/Consumer/PartitionContext.cs

```diff
@@ -32,7 +32,7 @@ public class PartitionContext
         ///   created with <see cref="ReadEventOptions.TrackLastEnqueuedEventProperties" /> set.
         /// </summary>
         ///
-        /// <returns>The set of properties for the last event that was enqueued to the partition.</returns>
+        /// <returns>The set of properties for the last event that was enqueued to the partition.  If no events were read or tracking was not set, the properties will be returned with default values.</returns>
```

Comment:
Orthogonal, but @ellismg found that this comment was based on a previous implementation and did not reflect the current behavior.